test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampProviderTests.cs

```diff
@@ -188,12 +191,11 @@ public async Task GetTimestampAsync_WhenLoggerNull_ThrowsAsync()
                     target: SignaturePlacement.PrimarySignature
                 );
 
-                // Act
-                Action timestampAction = async () => await timestampProvider.GetTimestampAsync(request, null, CancellationToken.None);
-
                 // Assert
-                timestampAction.ShouldThrow<ArgumentNullException>()
-                    .WithMessage(string.Format(ArgumentNullExceptionMessage, "logger"));
+                var exception = await Assert.ThrowsAsync<ArgumentNullException>(
+                    () => timestampProvider.GetTimestampAsync(request, null, CancellationToken.None));
```

Comment:
nit: add named parameters in all `timestampProvider.GetTimestampAsync()`

Reply:
Thanks! Fixed. Add named parameters for null value.